src/containers/Auth/LoginScreen.js

```diff
@@ -74,17 +75,17 @@ export const LoginScreen = () => {
         >
           <View style={styles.container}>
             <TextSourceSans style={styles.title}>
-              Login
+              {t('Login')}
             </TextSourceSans>
-            <Trans>
-              <TextSourceSans style={styles.registerText}>
+            <TextSourceSans style={styles.registerText}>
+              <Trans t={t}>
```

Comment:
is this prop needed?

Reply:
I  think it's needed to update the view if the language changes, if you just use <trans> it wouldn't. At this point it's probably more of a theoretical problem as you can't really change the language in that view